.gitignore

```diff
@@ -12,3 +12,4 @@ bin/
 vendor/
 # Ignores charts pulled for dependency build tests
 cmd/helm/testdata/testcharts/issue-7233/charts/*
+.pre-commit-config.yaml
```

Comment:
this intentional?

Reply:
Yer, it was. I use precommit for local gate keeping. I have a confi file which I want to ignore.

Reply:
It would have been cleaner in another PR

Reply:
I reverted the change and created https://github.com/helm/helm/pull/13465 PR